internal/provisioner/fluentbit.go

```diff
@@ -80,12 +79,38 @@ func (f *fluentbit) NewHelmDeployment(logger log.FieldLogger) *helmDeployment {
 	if err != nil {
 		logger.WithError(err).Error("unable to lookup private zone name")
 	}
+	domainNameSplitted := strings.Split(privateDomainName, ".")
+	sumoLogic := `[OUTPUT]
+	Name  forward
+	Match  *
+	Host  example.placeholder.mattermost.com
+	Port  %s
+	tls  On
+	tls.verify  Off
+	#tls.ca_file  /etc/certs/ca-chain.pem`
+
+	env := domainNameSplitted[1]
+	var sumoLogicConf string
+	if env == "prod" {
```

Comment:
Using a switch statement here would probably clean up the code quite a bit! https://tour.golang.org/flowcontrol/9

Reply:
Initially I tried with switch, but then it looked messier so I changed with a simple if/else statement

Reply:
Yeah, not a big deal at all, but you should be able to do something like this if you want:

```
var sumoLogicConf string
switch domainNameSplitted[1] {
case "prod":
	sumoLogicConf = fmt.Sprintf(sumoLogic, "24222")
case "staging":
	sumoLogicConf = fmt.Sprintf(sumoLogic, "24011")
case "test":
	sumoLogicConf = fmt.Sprintf(sumoLogic, "24022")
}
```

Reply:
Ahhh I see, I did something like:
```
	switch sumoLogicConf := domainNameSplitted[1]; sumoLogicConf {
	case "prod":
		sumoLogicConf := fmt.Sprintf(sumoLogic, "24222")
	case "staging":
		sumoLogicConf := fmt.Sprintf(sumoLogic, "24011")
	case "test":
		sumoLogicConf := fmt.Sprintf(sumoLogic, "24022")
	default:
		sumoLogicConf := " "
	}
```
I followed the wrong example probable
Thanks Gabe for the help!